src/uct/sm/mm/mm_pd.h

```diff
@@ -38,20 +38,25 @@ typedef struct uct_mm_mapper_ops {
 
     ucs_status_t (*query)();
 
+    size_t       (*get_path_len)(uct_pd_h pd);
```

Comment:
i would rename this to "get_path_size" because it would imply that it returns the total size of the buffer including the null terminator, not just the string length (like strlen)